Orange/widgets/unsupervised/owdistances.py

```diff
@@ -160,6 +161,8 @@ def checks(metric, data):
         except MemoryError:
             self.Error.distances_memory_error()
             return
+
+        met = np.nan_to_num(met)
```

Comment:
> perhaps len(data) * np.nanmax(data)

If you opt for this suggestion, please, just don't use `np.nanmax(data)`. We should never use `np.nanmax(data)` since it doesn't work on sparse data! We have our own implementation of `nanmax` in `Orange/statistics/util.py`.